docs/source/private_data_tutorial.rst

```diff
@@ -1083,6 +881,79 @@ If you go back and look at the block height in your logs, you will see that the
 due to hitting the fourth block (we only increased by two blocks in transferring the asset to Org2), but rather
 because when the asset transfer occurred, Org1 did not meet the ``read`` policy of the ``Org2MSPPrivateCollection``.
 
+
+.. _pd-purge:
+
+Purge Private Data
+------------------
+
+For use cases where private data only needs to be on the ledger until it can be
+replicated into an off-chain database, it is possible to "purge" the data after
+a certain set number of blocks, leaving behind only hash of the data that serves
+as immutable evidence of the transaction.
+
+There may be private data including personal or confidential
+information, such as the ``appraisedValue`` data in our example, that the transacting
+parties don't want disclosed to other organizations on the channel. Thus, it
```

Comment:
Private data in a collection is never disclosed to non-member peers, so I wouldn't say that is the reason to purge it.
Focus on the limited lifespan aspect.  The appraisedValue scenario is good, as you may want some time limit on the agreement (of course, number of blocks is a very rough way to specify time).

Reply:
The ``appraisedValue`` data in our example contains a private agreement that
the organization may want to expire after a certain period of time. Thus, it
has a limited lifespan, and can be purged after existing unchanged on the
blockchain for a designated number of blocks using the ``blockToLive`` property
in the collection definition.